.circleci/config.yml

```diff
@@ -75,9 +72,12 @@ jobs:
       - run:
           name: Install playwright
           command: npx playwright install
+      - run:
+          name: Install pnpm
+          command: npm install -g pnpm
```

Comment:
Just as a suggestion since we are using this in multiple places, probably we can have this and other shared `run` commands within a `common_steps` and extend it in all:

```
common_steps: &common_steps
  - checkout
  - restore_cache: *restore_yarn_cache
  - run:
      name: Install pnpm
      command: npm install -g pnpm


# later in jobs...
integration-tests:
    <<: *defaults
    docker:
      - image: mcr.microsoft.com/playwright:focal
    steps:
      <<: *common_steps
      - attach_workspace:
          at: ~/table
      - run:
          name: Install playwright
          command: npx playwright install

```
but if you think it's unnecessary at this step, feel free to ignore this comment 🙂

Reply:
We are moving to github actions in #1125 so I will not do any work to improve this. Just wanted it to sort of work